applitools.config.js

```diff
@@ -1,3 +1,5 @@
 module.exports = {
   serverUrl: 'https://twilioeyesapi.applitools.com',
+  batchId: process.env.CIRCLE_SHA1,
```

Comment:
Setting the batchID here instead of the circleCi conig (which is what the docs say to do but it's wrong)

Making it exit with `0` so the circle job doesn't error if there are differences, we want Applitools to error so that resolving the diffs in Applitools marks the checks as green. (you'd have to run circleCI again to get it to be green otherwise)